client/src/tutorials/tutorials.ts

```diff
@@ -98,4 +98,24 @@ export const TUTORIALS: TutorialData[] = [
     categories: [TutorialCategory.ANCHOR, TutorialCategory.GAMING],
     elementImport: () => import("./TinyAdventureTwo"),
   },
+
+  {
+    name: "Battle Coins",
+    description:
+      "Learn to create a token mint with metadata, mint tokens, and burn tokens. Defeat enemies to earn tokens and restore your health by burning tokens.",
+    imageSrc: getTutorialImgSrc("battle-coins/battleCoins.jpg"),
+    authors: [
+      {
+        name: "Jonas Hahn",
+        link: "https://twitter.com/solplay_jonas",
+      },
+      {
+        name: "John",
+        link: "https://twitter.com/ZYJLiu",
+      },
+    ],
+    level: TutorialLevel.BEGINNER,
```

Comment:
Given this tutorial has a lot more going than the other beginner tutorials, I think it's appropriate for this to be intermediate.

```suggestion
    level: TutorialLevel.INTERMEDIATE,
```